packages/scaffolding-cli/src/domain/workers/main_worker.ts

```diff
@@ -214,6 +214,38 @@ export class MainWorker {
             };
         }
     }
+
+    async js_testcafe_tfs(instructions: CliAnswerModel): Promise<CliResponse> {
+        let selectedFlowResponse: CliResponse = <CliResponse>{}
+        try {
+            let buildInput: Array<BuildReplaceInput> = js_testcafe.in_files(instructions.project_name)
+            
+            let new_directory: TempCopy = await Utils.prepBase(instructions.project_name)
+
+            await Utils.constructOutput(staticConf.js_testcafe.folder_map, new_directory.final_path, new_directory.temp_path)
+
+            let val_maps: Array<Replacetruct> = buildReplaceFoldersAndVals(new_directory.final_path, buildInput);
+
+            await Utils.valueReplace(val_maps)
+            if (instructions.create_config) {
+                await Utils.writeOutConfigFile(`${instructions.project_name}.bootstrap-config.json`, instructions)
```

Comment:
I would get rid of the if here and jsut writeout config - as that is. what is happening in the rest of the methods now